src/caselawclient/Client.py

```diff
@@ -103,21 +127,29 @@ def _path_to_request_url(self, path: str) -> str:
     def _court_list_splitter(self, court_text: str) -> Set[str]:
         return set(court_text.lower().replace(" ", "").split(","))
 
-    def _court_list(self, court_text: str) -> Optional[list[str]]:
-        if not court_text.strip():
-            return None
-        alt_names = {
+    def _join_court_text(self, court_text: str) -> str:
```

Comment:
only new thing for the court_list method is `_join_court_text` which has been taken from public_ui's usage of `api_client.advanced_search`

`https://github.com/nationalarchives/ds-caselaw-public-ui/blob/6c966bba962e3a8efdd951029b68fa9cb576b3e5/judgments/utils.py#L42`